packages/__docs__/src/Document/index.tsx

```diff
@@ -199,6 +200,28 @@ class Document extends Component<DocumentProps, DocumentState> {
     )
   }
 
+  renderThemeLink() {
+    const { srcUrl, srcPath, legacyGitBranch } = this.props.doc
+    if (!srcUrl) return
+
+    const themeUrl = srcUrl.replace('index.tsx', 'theme.ts')
+
+    const themePath = srcPath.replace('index.tsx', 'theme.ts')
+
+    let legacySrcUrl
+
+    if (legacyGitBranch) {
+      legacySrcUrl = themeUrl.replace('/master/', `/${legacyGitBranch}/`)
+    }
+
+    return (
+      <View as="div" margin="0 0 x-large 0">
+        You can check these variables at{' '}
+        <Link href={legacySrcUrl || themeUrl}>{themePath}</Link>
```

Comment:
I would also use this link in the block below on the "default theme variables" text.

<img width="1006" alt="Screenshot 2022-09-20 at 10 10 45" src="https://user-images.githubusercontent.com/11446033/191204053-adfab29b-d6a3-471a-9d8f-b15dabdfd9fb.png">